test/fail_compilation/ice16575.d

```diff
@@ -0,0 +1,27 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ice16575.d(17): Error: function `ice16575.t11` parameter `a` of `extern(C++)` function cannot be of type `char[]`
```

Comment:
`function` is redundant because the error is generated for the function, which i didn't notice.